src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpHandlerToFilter.cs

```diff
@@ -0,0 +1,262 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Whatever naming convention we adopt for the other files of this nature, it needs to apply to all of them.  HttpHandlerToFilter.netcore50.cs?